.circleci/config.yml

```diff
@@ -82,6 +82,14 @@ workflows:
             branches:
               only:
                 - main
+  prod_deploy:
+    jobs:
+      - lint_and_test
+          filters:
```

Comment:
I think we need to add prod_deploy:

Reply:
oops ... forgot to push my fix. it's missing a ':' after `lint_and_test`